### ISSUE TITLE
Document the build process of the Windows application

### ISSUE THREAD

**Issue by AndreasFreund** [amosproj/amos2024ss07-updating-flash-boot-loader#22]
## User story
1. As a developer
2. I need to have documentation on the build process
3. So that I can build the Windows application with the GUI easily

## Acceptance criteria
* Use Markdown file if possible
* Guide to setup development environment
* Guide to build the application after setting up the development environment

## Definition of done (DoD)
* Added only after week 5
* The same for all features
* Here goes the project specific part

## DoD general criteria
* Feature has been fully implemented
* Feature has been merged into the mainline
* All acceptance criteria were met
* Product owner approved features
* All tests are passing
* Developers agreed to release
